src/scripts.js

```diff
@@ -132,4 +141,20 @@ function checkUserInputs() {
     } else {
       return res.json()
     }
+  }
+
+  function checkUserCredentials() {
```

Comment:
Another great example of error handling! I didn't know you could use split with a string